scheduler/CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file
  
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.14.0] - 2018-03-28
+### Added
+- Support for listing custom executor jobs in /list endpoint, from @dposada
+- Kill instances for cancelled jobs on leadership election, from @pschorf
+### Changed
+- Performance improvements, from @scrosby
```

Comment:
Should we be more specific about the areas improved?

Reply:
Done